src/microbe_stage/MicrobeStage.cs

```diff
@@ -514,6 +514,21 @@ public override void OnReturnFromEditor()
         {
             tutorialGUI.EventReceiver?.OnTutorialDisabled();
         }
+        else
+        {
+            // Show day/night cycle tutorial when entering a patch with sunlight
+            if (GameWorld.WorldSettings.DayNightCycleEnabled)
+            {
+                var sunlight = SimulationParameters.Instance.GetCompound("sunlight");
+                var patchSunlight = GameWorld.Map.CurrentPatch!.Biome
+                    .GetCompound(sunlight, CompoundAmountType.Biome).Ambient;
+
+                if (patchSunlight > 0.01f)
+                {
+                    TutorialState.SendEvent(TutorialEventType.MicrobePlayerEnterSunlightPatch, EventArgs.Empty, this);
```

Comment:
Might need to add a logic for automatic closing for the tutorial window after a certain interval.

Reply:
I agree, and the timer should be in the tutorial class itself.

Reply:
Ok, I'll add that